image/templates/helm/stackrox-secured-cluster/templates/_init.tpl.htpl

```diff
@@ -34,19 +34,22 @@
     $.Values, we apply some bootstrap defaults.
    */}}
 {{ $rox := deepCopy $values }}
-
+{{ $_ := set $ "_rox" $rox }}
 
 {{ $configShape := $.Files.Get "internal/config-shape.yaml" | fromYaml }}
-
-{{/* Only merge scanner config shapes if feature flag is enabled and kubectl output is disabled */}}
-[< if (not .KubectlOutput) ->]
+{{/* Only merge scanner config shapes if kubectl output is disabled */}}
+[<- if not .KubectlOutput >]
   {{ $configShapeScanner := $.Files.Get "internal/scanner-config-shape.yaml" | fromYaml }}
-  {{ $_ := include "srox.mergeInto" (list $rox $configShape $configShapeScanner) }}
-[< else >]
-  {{ $_ := include "srox.mergeInto" (list $rox $configShape) }}
-[< end >]
+  {{ $_ = include "srox.mergeInto" (list $configShape $configShapeScanner) }}
+  {{/* Only merge scanner-v4 config shapes if kubectl output is disabled and feature flag is enabled */}}
```

Comment:
nit: `scannerV4` to match the config-shape?

Reply:
Are you only referring the comment here?